src/lib/core/style/_noop-animation.scss

```diff
@@ -0,0 +1,9 @@
+// If the mat-animation-noop class is present on the components root element,
+// prevent non css animations from running.
+@mixin noop-animation() {
```

Comment:
`mat-noopable-animation`?

Reply:
I would also prefix it with an underscore